idaes/commands/tests/test_commands.py

```diff
@@ -633,6 +633,7 @@ def test_conf_file_paths(runner):
         os.remove(fname)
 
 
+@pytest.mark.skipif(sys.version_info[:2] == (3, 11), reason="Fails on Python 3.11")
```

Comment:
Do we have an issue to track this problem?

Reply:
Good point: #1140